pkg/antctl/command_definition.go

```diff
@@ -566,15 +572,21 @@ func (cd *commandDefinition) collectFlags(cmd *cobra.Command, args []string) map
 	}
 	if cd.namespaced() {
 		argMap["namespace"], _ = cmd.Flags().GetString("namespace")
+		if len(argMap["name"]) > 0 && len(argMap["namespace"]) == 0 {
+			return nil, fmt.Errorf("BadRequest: Please check the args for %s", cd.use)
```

Comment:
Please make the error be more specific, indicate how to correct the request "You must specify the namespace of resource to get".

Reply:
Fix this in command_message.go.